statsmodels/stats/contingency_tables.py

```diff
@@ -1402,3 +1484,1238 @@ def cochrans_q(x, return_object=True):
         return b
 
     return q_stat, pvalue, df
+
+
+def shift_zeros(dataframe):
+    """
+    chi-squared tests can't handle having expected counts of zero. so if the table contains
+    a cell with zero observations, jitter it up by 0.5
+    :param dataframe:
+    :return:
+    """
+    if dataframe.min().min() == 0:
+        return dataframe + 0.5
+    else:
+        return dataframe
+
+
+class MRCVTable(object):
```

Comment:
the name is not informative enough, it's difficult (impossible) to remember what MRCV stands for